host-controller/src/main/java/org/jboss/as/host/controller/discovery/DiscoveryOption.java

```diff
@@ -52,6 +52,7 @@
 
     /**
      *  Gets the host name of the remote domain controller.
+     *  This method may return null.
```

Comment:
I was not sure about this. It looks to me it's not expected to return null though.